multiqc/utils/search_patterns.yaml

```
@@ -16,6 +16,9 @@ anglerfish:
 bakta:
   fn: "*.txt"
   contents: "Bakta:"
+bamdst/coverage:
+  contents: "## The file was created by bamdst"
+  num_lines: 1
```

In case of unexpected crap from elsewhere
```suggestion
  num_lines: 5
```